web/profiles/joinup/joinup.behat.inc

```diff
@@ -2402,59 +2402,6 @@ JS;
     }
   }
 
-  /**
-   * Asserts that a user is subscribed to specific bundles in a collection.
-   *
-   * @param string $user
-   *   The user name.
-   * @param string $bundles
-   *   A list of bundles separated by commas.
-   * @param string $type
-   *   Either 'content' or 'rdf' which corresponds to 'node' and 'rdf_entity'
-   *   entity types respectively.
-   * @param string $group_label
-   *   The group name.
-   * @param string $group_bundle
-   *   The group bundle.
-   *
-   * @throws \Exception
-   *   Throws exception if invalid data passed as arguments or if the subscribed
-   *   bundles for the given type do not match the expected ones.
-   *
-   * @Given :user should be subscribed to :bundles :type bundles in the :group_label :group_bundle
-   */
-  public function shouldBeSubscribedToContentInCollection($user, $bundles, $type, $group_label, $group_bundle) {
```

Comment:
This is a duplicate of `\SubscriptionSubContext::assertCollectionContentSubscriptions()`